### DISCUSSION TITLE
Document what diagrams `fontSize` works with

### DISCUSSION THREAD

**Discussion by PaulRBerg** [mermaid-js/mermaid#4244] [General]
I have spent a few hours trying to understand how the [`fontSize`](https://github.com/mermaid-js/mermaid/blob/e4d2118d4bfa023628a020b7ab1f8c491e6dc523/packages/mermaid/src/config.type.ts#L36) config option works, only to find out that it doesn't work with flowcharts:

https://github.com/mermaid-js/mermaid/issues/2896

My documentation request is to add an explainer in your [docs website](https://mermaid.js.org/intro/) to clarify what diagrams `fontSize` can be used with.

**Comment by huynhicode** [comment]
Hi @PaulRBerg 

There are two ways of customizing themes: site-wide and to individual diagrams.

To set the font size to an individual diagram, modify the `themeVariable`by setting `fontSize` to the desired size.  The default is 16px.

Here is a link to our documentation on [Theme Configuration](https://mermaid.js.org/config/theming.html#theme-configuration).

![Screenshot 2023-03-24 at 10 13 34 AM](https://user-images.githubusercontent.com/35910788/227594957-8c7c9580-8f0c-4743-aac1-2d7d76ed5a65.png)

Here is an example from [Mermaid.Live](https://mermaid.live/edit#pako:eNpVkMFOhEAMhl9l0pMmkOjBCwcTF3QvazTZjQeBQ4UOMxGmk2HIugLv7iCa7PbU9vv_tukIFdcECciWj5VC58UhK4wI8ZCnyuned9iXIo7vpy150bGh0yQ2V1sWvWJrtWmuV_1mEYl03C0yEl5p8zmvKP31vxiaRJbv0Hq25Tk5HHkSj7l-VWH8JVGOguspl5hIjCt0IkVXQgQduQ51HU4fF0MBXlFHBSQhrUni0PoCojP0hk7jR0v9ohnXJQVINn6vv_-Mt3c39quABYbT57AGB8_7k6kg8W6gCAZbo6dMY-Ow-29aNO_MoZTY9qGmWnt2z-tnKzZSNzD_AAX_dVA)( (screenshot below) where I have set the `fontSize` of my `flowchart` to `150px`.

![screencapture-mermaid-live-edit-2023-03-24-10_01_17](https://user-images.githubusercontent.com/35910788/227592487-59e9f718-1d92-4751-b468-1da8e6d7bb4c.png)

Also, if you haven't joined already, there is a [Slack workspace](https://mermaid-talk.slack.com/) where you can post questions or see if someone else has posted a similar question for a quicker resolution.

**Reply by PaulRBerg** [reply]
Thanks for your answer, @huynhicode.

> There are two ways of customizing themes: site-wide and to individual diagrams.
>
> To set the font size to an individual diagram

It's still unclear from this if it is possible to customize the font size of all flow charts. From [issue #2896](https://github.com/mermaid-js/mermaid/issues/2896), I would be inclined to believe that the answer is "no," but I am not sure.

> there is a [Slack workspace](https://mermaid-talk.slack.com/) where you can post questions

Thanks, but I would prefer to keep chatting here so I don't have to sign up on another Slack workspace.
